### PR TITLE
Review Jenkinsfile for building stable branch

### PR DESCRIPTION
**Pull request by cristianonicolai**
This should only rely on published artifacts

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -18,26 +18,6 @@ pipeline {
                 sh 'printenv'
             }
         }
-        stage('Build kogito-runtimes') {
-            steps {
-                dir("kogito-runtimes") {
-                    script {
-                        githubscm.checkoutIfExists('kogito-runtimes', "$CHANGE_AUTHOR", "$CHANGE_BRANCH", 'kiegroup', "$CHANGE_TARGET")
```
Comment (evacchi) [inline Jenkinsfile]: maybe we can fix without removing by putting an if in front of this: `if (CHANGE_BRANCH != 'master' && CHANGE_TARGET != 'master') ` same down below

**Review by evacchi — CHANGES_REQUESTED** [review]
good point! although as mentioned in the other thread, I think of the stable branch as a moving pointer so that never diverges from master. WDYT?

File: Jenkinsfile (hunk 1 above)
Comment (cristianonicolai) [reply Jenkinsfile]: I think is actually `if($CHANGE_TARGET != "stable")`, changes targetting any other branch should still work with full build.
Will test PR here, if it works, I will submit to master and others branches...

**Comment by cristianonicolai** [comment]
> good point! although as mentioned in the other thread, I think of the stable branch as a moving pointer so that never diverges from master. WDYT?

true, that should make things easier to keep branches in sync.

**Pull request closed without merging**